src/app/page.js

```diff
@@ -11,54 +12,61 @@ export default function Home() {
   if (counter == 3) {
     document.getElementById("rocket").classList.add("animate-spin");
   }
-  
+
   const handleStop = () => {
     setCounter(0);
     document.getElementById("rocket").classList.remove(["animate-spin"]);
   };
 
   return (
-    <main className="flex min-h-screen min-w-screen flex-col items-center justify-center p-12 xl:p-0 md:text-2xl bg-blue-100">
-      <div className="drop-shadow-xl flex-col flex gap-10 items-center ">
-        <Player />
-        <div>
-          <h1
-            className="text-center font-black text-3xl md:text-5xl break-keep break-words mb-4 cursor-pointer"
-            onClick={headingClicked}
-          >
-            HackOdisha 3.0
-          </h1>
-          <h2 className="text-center font-black text-xl md:text-3xl break-keep break-words">
-            Coming Soon{" "}
-           <span className="group">
-           <p
-              id="rocket"
-              className="cursor-pointer inline-block"
-              onClick={handleStop}
+    <>
+      <Hero />
```

Comment:
Thanks for creating the Hero component but it may stay commented for the time being, that is, before anyone works on the component.